src/block_cache/block_chunkcache.c

```diff
@@ -153,12 +187,13 @@ __chunkcache_free_chunk(WT_SESSION_IMPL *session, WT_CHUNKCACHE_CHUNK *chunk)
     if (chunkcache->type == WT_CHUNKCACHE_IN_VOLATILE_MEMORY)
         __wt_free(session, chunk->chunk_memory);
     else {
-#ifdef ENABLE_MEMKIND
-        memkind_free(chunkcache->memkind, chunk->chunk_memory);
-#else
-        __wt_err(session, EINVAL,
-          "Chunk cache requires libmemkind, unless it is configured to be in DRAM");
-#endif
+        /* Update the bitmap to show free chunk, then free the memory of the chunk */
+        size_t index = (size_t)(chunk->chunk_memory - chunkcache->memory) / chunkcache->chunk_size;
+        bit = (uint8_t *)&chunkcache->bitmap[index / 8];
+        ret = __wt_atomic_cas8(bit, *bit, *bit & ~(0x01 << (index % 8)));
```

Comment:
`0x01 << (index % 8) ` should I put this in a macro as well under "shift_one" or similar?